lib/temporal/activity/poller.rb

```diff
@@ -66,6 +66,11 @@ def poll_for_task
         client.poll_activity_task_queue(namespace: namespace, task_queue: task_queue)
       rescue StandardError => error
         Temporal.logger.error("Unable to poll activity task queue: #{error.inspect}")
+
+        Temporal.configuration.error_handlers.each do |handler|
+          handler.call(error, nil)
```

Comment:
We'll also need to put guards around call to the provided lambdas, we can't afford crashing on buggy error handling